### PR TITLE
Change the renumber_sampled_edgelist function behavior.

### PR DESCRIPTION
**Pull request by seunghwak**
There was a misalignment between the `renumber_sampled_edgelist` function behavior and what PyG and DGL need.

This PR fixes this.

### CODE REVIEW

**Comment by alexbarghi-nv** [comment]
@seunghwak is this PR ready yet?  I'm still seeing the wrong behavior after trying sampling with the PR.

File: cpp/src/sampling/renumber_sampled_edgelist_impl.cuh [hunk 1]
```diff
@@ -324,121 +134,329 @@ compute_renumber_map(raft::handle_t const& handle,
...
-      label_dst_triplet_first + (*unique_label_dst_pair_label_indices).size(),
-      thrust::make_zip_iterator(
-        merged_label_indices.begin(), thrust::make_discard_iterator(), merged_vertices.begin()));
-
-    return std::make_tuple(std::move(merged_vertices), std::move(merged_label_indices));
+    renumber_map_label_indices.reserve(renumber_map.capacity(), handle.get_stream());
+
+    auto num_chunks = (edgelist_srcs.size() + (approx_edges_to_sort_per_iteration - 1)) /
+                      approx_edges_to_sort_per_iteration;
+    auto chunk_size = (num_labels + (num_chunks - 1)) / num_chunks;
```
Comment (alexbarghi-nv) [inline cpp/src/sampling/renumber_sampled_edgelist_impl.cuh:271]:
```suggestion
    auto chunk_size = (num_labels + (num_chunks - 1)) / num_chunks;
    if(num_chunks < 1) {
      num_chunks = 1;
    }
```

**Review by alexbarghi-nv — CHANGES_REQUESTED** [review]
Looks good, just the issue with chunk size.  Benchmarks succeed when ensuring `num_chunks` is >=1

File: cpp/src/sampling/renumber_sampled_edgelist_impl.cuh (hunk 1 above)
Comment (alexbarghi-nv) [reply cpp/src/sampling/renumber_sampled_edgelist_impl.cuh]: This has been fixed now.

**Review by alexbarghi-nv — APPROVED** [review]
👍

**Review by ChuckHastings — APPROVED** [review]

**Pull request merged**